nixos/tests/installer.nix

```diff
@@ -74,7 +74,7 @@ let
       throw "Non-EFI boot methods are only supported on i686 / x86_64"
     else ''
       def assemble_qemu_flags():
-          flags = "-cpu host"
+          flags = "-cpu host,-vmx"
```

Comment:
not sure if this is a problem, but this passes `-vmx` also on non-x86 platforms. Does that even work then?

Reply:
Have to check that one, thanks!